### PR TITLE
chore: Update yarn.lock for yarn 3.8.7

### PR DESCRIPTION
**Pull request by mareklibra**
This is follow-up for updating the yarn to 3.8.7 #192

### CODE REVIEW

**Review by gciavarrini — APPROVED** [review]
/lgtm

**Comment by christoph-jerolimov** [comment]
I can confirm that I got the same `yarn.lock` change after removing and reinstall node_modules first on the root level and then in a workspace.

Merge this after we have already another lgtm.

Thanks @mareklibra and @gciavarrini

**Review by christoph-jerolimov — APPROVED** [review]

**Pull request merged**
